src/Components/Reports/Util.js

```diff
@@ -0,0 +1,68 @@
+import { get } from 'lodash';
+import { pdfRowKeys, percentageKeys, reportRowKeys, SYSTEMS_REPORT_FILE_NAME } from '../../constants';
+
+export const formatData = (data, type) => {
+
+    const systemsRowsData = [];
+    const rowKeys = type === 'json' ?  reportRowKeys : pdfRowKeys;
+
+    data.map((systemItem) => {
+        let rowData = type === 'json' ? {} : [];
+
+        rowKeys.map((rowKey) =>{
+            let rowValue =  get(systemItem, rowKey, '');
+            rowValue = (rowValue === null || rowValue === -1) ?  'N/A' : rowValue.toString();
+            rowValue = (rowValue !== 'N/A' && percentageKeys.includes(rowKey)) ? `${rowValue}%` : rowValue;
+
+            if (type === 'json') {
+                rowData[rowKey] = rowValue;
+            } else if (type === 'pdf') {
+                rowData.push(rowValue);
+            }
+        });
+
+        systemsRowsData.push(rowData);
+    });
+
+    return systemsRowsData;
+
+};
+
+export const responseToJSONData = (data) => {
+    const systemsRowsData = formatData(data, 'json');
+    return JSON.stringify(systemsRowsData);
+};
+
+export const responseToCSVData = (data) => {
+    const items =  formatData(data, 'json');
+    const replacer = (key, value) => value === null ? 'N/A' : value;
+    const header = Object.keys(items[0]);
+    const csvData = [
+        header.join(','), // header row first
+        ...items.map(row => header.map(fieldName => JSON.stringify(row[fieldName], replacer)).join(','))
+    ].join('\r\n');
+
+    return csvData;
+};
+
+export const generateFilterText = (filters) => {
+    let filterText  = '';
+    const hasStateFilter = filters?.stateFilter?.length > 0;
+    const hasNameFilter =  filters?.hostnameOrId?.length > 0;
+
+    if (hasStateFilter || hasNameFilter) {
+        filterText = `\nFilters applied\n`;
+        filterText = hasStateFilter ? filterText.concat(`State: ${filters.stateFilter.toString()}\t\t\t\t\t`) : filterText;
```

Comment:
Don't know why `\t` not working. 

![Screenshot from 2022-04-01 19-20-00](https://user-images.githubusercontent.com/61837065/161283455-ff998417-9bad-4d7f-aec3-34580a9fc08b.png)

![Screenshot from 2022-04-01 19-26-03](https://user-images.githubusercontent.com/61837065/161283502-40a87d1e-7c9f-4950-82cc-ed90e665d0ef.png)

Reply:
hmm weird @r14chandra. Can you take pull with latest changes and check again please?

Reply:
@r14chandra as you mentioned I checked in firefox and you are correct the /t is not working there. I have asked Stefan if its okay to show each filter on seperator line (since we can have many values for OS) based on his inputs I will update the code accordingly 🙌🏻

Reply:
@r14chandra Stefan [replied regarding using \n for text seperator](https://issues.redhat.com/browse/RHIROS-514) and mentioned its okay to do that. I have updated the code based on that and have pushed the changes 💯